routes/api.php

```diff
@@ -13,12 +13,13 @@
 |
 */
 
-Route::post('/register','Auth\RegisterController@register')->name('register'); 
-Route::post('/login', 'Auth\LoginController@login')->name('login'); 
-Route::post('/logout', 'Auth\LoginController@logout')->name('logout'); 
+Route::post('/register','Auth\RegisterController@register')->name('register');
+Route::post('/login', 'Auth\LoginController@login')->name('login');
+Route::post('/logout', 'Auth\LoginController@logout')->name('logout');
 Route::get('/user', function(){return Auth::user();})->name('user');
 
-Route::get('/job', 'JobController@index')->name('jobget'); 
+Route::get('/job', 'JobController@index')->name('jobget');
+Route::get('/job/{job_id}', 'JobController@show')->name('job.show');
```

Comment:
```suggestion
Route::get('/job/{id}', 'JobController@show')->name('job.show');
```
自分もURL間違えてました！
{job_id} -> {id}